app/controllers/concerns/verify_profile_concern.rb

```diff
@@ -10,6 +10,8 @@ def account_or_verify_profile_url
 
   def profile_needs_verification?
     return false if current_user.blank?
+    return false if sp_session[:ial2_strict] &&
+                    !IdentityConfig.store.usps_upload_allowed_for_strict_ial2
```

Comment:
is it too late to rename this feature flag? we switched everything using the mail-in flow to be called "GPO" to differentiate from IPP @ USPS https://github.com/18F/identity-idp/pull/4837